src/js/actions/guides.js

```diff
@@ -313,6 +313,36 @@ define(function (require, exports) {
     setGuide.transfers = [resetGuidePolicies, deleteGuide];
 
     /**
+     * Clears all the guides in the given document
+     *
+     * @param {{id: number}} document Document model or an object containing document ID
```

Comment:
I was matching the other guide functions (where we receive the document ID from Photoshop and don't bother to grab the model), but this can be changed for clearGuides since it's an action and not just a handler.